app/konnect/analytics/index.md

```diff
@@ -25,52 +27,53 @@ routes, the graphs show requests broken down by status codes.
-
-Interval | Description  
-------|----------|
-Last 15 minutes | Data is aggregated in ten second increments.
-Last hour| Data is aggregated in one minute increments.
-Last three hours | Data is aggregated in one minute increments.
-Last six hours | Data is aggregated in ten minute increments.
-Last 12 hours| Data is aggregated in ten minute increments.
-Last 24 hours| Data is aggregated in ten minute increments.
-Last seven days | Data is aggregated in one hour increments.
-Last 30 days | Data is aggregated in one hour increments.
-
-{:.important}
-> Free tier users can only select intervals up to 24 hours.
-
-## Terms
-
-**Request Count**
-: Displays a count of all proxy requests received. This includes requests that
-were rejected due to rate limiting, failed authentication, and so on.
-
-**Status Codes**
-: Displays visualizations of cluster-wide status code classes (1xx, 2xx, 3xx,
-  4xx, 5xx). The Status Codes view contains the counts of status code classes
-  graphed over time, as well as the ratio of code classes to total requests.
-
-**Time frame selector**
-: Controls the time frame of data visualized, which indirectly controls the
-granularity of the data. For example, the “5M” selection displays 5 minutes in
-1-second resolution data, while longer time frames display minute, hour, or
-days resolution data.
-
-**Traffic metrics**
-: Provide insight into which of your services and service versions are being
-used and how they are responding.
+From {% konnect_icon analytics %} Analytics, you can view dashboards, access historical data for a range greater than 30 days, and customize the entities in a report:
+* View the Analytics summary dashboard to track traffic, errors by error code, and latency across all services in your organization.
+* Create a [custom report](/konnect/analytics/use-cases/) for any number of services, service versions, routes, applications, or status codes.
+You can export any custom report as a CSV file.
+
+The summary dashboard provides metrics for services cataloged by Service Hub within a selected time interval for the following categories:
+
+* **Traffic**: Total number of HTTP requests.
+* **Errors**: Percentage of failed HTTP requests.
+* **Latency**: Request latency at P50, P95, and P99 percentiles.
+
+These categories measure trends by comparing metrics across fixed comparable time intervals and plotting the data points. For example, hour-over-hour, day-over-day, week-over-week, and month-over-month.
+
+You can view a graph for each category by clicking **Traffic**, **Errors**, or **Latency**, and switching between the views.
+
+* **Traffic**: This graph displays the total number of HTTP requests categorized by successful and failed requests over the specified time interval.
+    * Successful requests contain all requests that returned a **1xx-3xx** status code.
+    * Failed requests contain all requests that returned a **4xx-5xx** status code.
+    * Custom 6xx status codes are not tracked.
```

Comment:
It might be better to say "out of spec" rather than "custom" -- or something like "status codes outside of this range are not reported".  Maybe it's just me and I've spent too much time in this area, but I feel like "custom" usually has a more positive connotation and I really don't want anyone to get the impression that nonstandard status codes are a good idea.